README.md

```diff
@@ -42,7 +42,8 @@ Relies on the [ocdeployer](https://github.com/bsquizz/ocdeployer) tool. This too
 deploys the resulting openshfit templates into specified cluster. 
 
 ~~~bash
-ocdeployer deploy -t openshift patchman-engine-ci -s build,deploy --secrets-local-dir openshift/secrets -e ./openshift/ci-env.yml
+cd e2e-deploy
```

Comment:
Please remove this information, as it's not public. Lets just keep instructions for deploying locally here